src/rgw/rgw_rest_s3.cc

```diff
@@ -6613,6 +6636,11 @@ int RGWSelectObj_ObjStore_S3::handle_aws_cli_parameters(std::string& sql_query)
   extract_by_tag("QuoteEscapeCharacter", m_escape_char);
   extract_by_tag("CompressionType", m_compression_type);
   extract_by_tag("Enabled", m_enable_progress);
+  extract_by_tag("FieldDelimiter", output_column_delimiter);
```

Comment:
the tag "FieldDelimiter" is been used twice for  m_column_delimiter and output_column_delimiter